### ISSUE TITLE
Posting links to youtube videos breaks the link and shows extra html code in the text

### ISSUE THREAD

**Issue by ExtReMLapin** [gmodstore/gmodstore-issues#241]
It wasn't a "migrating"/"markdown" issue, when linking youtube videos it just turn the link into stuff like that : 

from this : 

![](https://i.imgur.com/5ftfr6f.png)

to : 

![](https://i.imgur.com/UyuQIyU.png)

**Comment by ExtReMLapin** [comment]
probably because posting youtube links turn the "text" into a video if you see what i mean

**Comment by ExtReMLapin** [comment]
temp fix is to post a youtu.be link

**Comment by Marcuzz** [comment]
It works fine, you need to make sure it's not turned in to a link. If it's turned in to a link it doesn't know how to treat it

**Comment by ExtReMLapin** [comment]
Yeah but what if i want to make a text linking to a youtube video lol


unless i misunderstood your message

**Comment by Marcuzz** [comment]
Yeah that's what you can't do right now. I'll see if I can fix it with some regex
